components/tyk-mdcb/values.yaml

```diff
@@ -233,7 +237,7 @@ mdcb:
     # readiness includes details about readiness probe used in MDCB Deployment.
     readiness:
       # path represents the http path to be used in readiness probe in MDBC deployment.
-      path: "/health"
+      path: "/readiness"
```

Comment:
am wondering if in https://github.com/TykTechnologies/tyk-charts/pull/306/files#diff-b8f26e71ad3157bc8ed4f570bcc62cabdd9a80e016ad01e39db77137cb0c67a5R50
we should do something like:

`path: {{- if  semverCompare ">=2.6.0" $version }}/liveness{{ else }}/health{{ end }}`

Reply:
it's nice if we do it for backward compatibility